index.html

```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
                             <li><a href="#"><img src="assets/img/angies.png" alt="angies" /></a></li>
+                            <li><a href="#"><img src="assets/img/twitter.png" alt="twitter" /></a></li>
+                            <li><a href="#"><img src="assets/img/medium.png" alt="medium" /></a></li>
                         </ul>
                     </div>
+                    <div class="read-more">
+                        <a href="#">
+                            Get my resume
+                        </a>
+                    </div>
+                </div>
+                <div>
+                    <ul>
+                        <li class="categories open">
+                            <a href="#"><span>Languages</span><span><img src="./assets/img/arrow-down.png"
+                                        alt="more" /></span></a>
+                            <div class="item-list">
+                                <div class="item">
+                                    <div class="logo">
+                                        <img src="./assets/img/js.svg" alt="javasript" />
+                                    </div>
+                                    <div>
+                                        Javascript
+                                    </div>
+                                </div>
+                                <div class="item">
+                                    <div class="logo">
+                                        <img src="./assets/img/html.svg" alt="html" />
+                                    </div>
+                                    <div>
+                                        HTML
+                                    </div>
+                                </div>
+                                <div class="item">
+                                    <div class="logo">
+                                        <img src="./assets/img/css.svg" alt="css" />
+                                    </div>
+                                    <div>
+                                        CSS
+                                    </div>
+                                </div>
+                            </div>
+                        </li>
+                        <li class="categories">
+                            <a href="#">
+                                <span>
+                                    Frameworks
+                                </span>
+                                <span><img src="./assets/img/arrow-right.png" alt="more" /></span>
+                            </a>
+                        </li>
+                        <li class="categories">
+                            <a href="#"><span>Skills</span><span><img src="./assets/img/arrow-right.png"
+                                        alt="more" /></span></a>
+                        </li>
+                    </ul>
                 </div>
             </section>
```

Comment:
I suggest you put this section inside the "works" section as it is a project requirement that you create just the work and about section in this milestone and wrap other contents inside them🤗🤗🤗